cfme/containers/provider/__init__.py

```diff
@@ -27,27 +27,27 @@
         'containers_provider_new':
             lambda _: cfg_btn('Add a New Containers Provider'),
         'containers_provider':
-        [
-            lambda ctx: sel.check(Quadicon(ctx['provider'].name, None).checkbox),
-            {
-                'containers_provider_edit':
-                lambda _: cfg_btn('Edit Selected Containers Provider'),
-                'containers_provider_edit_tags':
-                lambda _: pol_btn('Edit Tags')
-            }],
+            [
```

Comment:
I'm not so worried about this at the moment as all of this is the old navigate and we are getting rid of that soon anyway, so I'd ctually be more inclined to say, we _can_ merge this, but soon it will no longer work.